fe/fe-core/src/main/java/com/starrocks/master/MasterImpl.java

```diff
@@ -309,10 +309,10 @@ private void finishCreateReplica(AgentTask task, TFinishTaskRequest request) {
                         task.getBackendId() + ": " + request.getTask_status().getError_msgs().toString());
```

Comment:
if tablet create failed, we should consider delete the shard for staros tablet. I suggest add a TODO comment for this.

Reply:
Ok, i add a TODO first.